packages/calcite-components/src/components/navigation-logo/navigation-logo.scss

```diff
@@ -10,27 +27,30 @@
     cursor-pointer
     transition-default
     focus-base
-    no-underline
-    text-0h;
-  color: inherit;
-  border-block-end: 2px solid transparent;
+    no-underline;
+  color: var(--calcite-navigation-logo-text-color, inherit);
+  font-size: var(--calcite-font-size-md);
+  line-height: var(--calcite-font-line-height-fixed-lg);
+  border-block-end: var(--calcite-spacing-base) solid
+    var(--calcite-navigation-logo-background-color, var(--calcite-color-foreground-1));
 }
 
 .anchor:hover,
 .anchor:focus {
-  @apply bg-foreground-2;
+  background-color: var(--calcite-navigation-logo-background-color-hover, var(--calcite-color-foreground-2));
```

Comment:
in a case like this you can do one of two things. 
1. set the background color on the base `.anchor` as `background-color: var(--calcite-navigation-logo-background-color, var(--calcite-internal-navigation-logo-background-color), transparent);` and then reassign the internal var `--calcite-internal-navigation-logo-background-color` when `.anchor:hover, .anchor:focus {--calcite-internal-navigation-logo-background-color: var(--calcite-color-foreground-2); }`

2. set style props based on the :host state. `:host(:hover), :host(:focus) { .anchor { background-color: var(--calcite-navigation-logo-background-color-hover, var(--calcite-color-foreground-2)); }}`

3. use state props 
```
.anchor:hover,
.anchor:focus {
  background-color: var(--calcite-navigation-logo-background-color-hover, var(--calcite-color-foreground-2));
```